File: Problem 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //prompting them for a positive integer between 1 and 8, inclusive; re-prompt until right
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    //create pyramid
    for (int rows = 0; rows < height; rows++)
    {
        for (int spaces = 0; spaces < height -rows-1; spaces++)
        {
            printf(" ");
        }
        for (int hash = height - rows - 1; hash < height; hash++)
        {
            printf("#");
        }
        printf("  ");
        for (int hash = height - rows - 1; hash < height; hash++)
        {
            printf("#");
        }
        printf("\n");
    }


}

File: Problem 6/sentimental-mario-more/mario.py
from cs50 import get_int

# TODO
while True:
    height = get_int("Height: ")
    if (height >= 1 and height <= 8):
        break
space = " "
hashs = "#"
for i in range(1, height+1, 1):
    print((space*(height - i)) + (hashs*i) + (space*2) +  (hashs*i))